index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>hello microverse</h1>
```

Comment:
- [ ] As stated in the project requirement your` h1` tag has A .css file that includes `one class` with styles for your header (make it in your favorite color) Just like the below sample 
-```
   #html
<h1class="heading">hello microverse</h1>
```

-```
 #css
heading {
  color: aqua;
}
```